build.gradle

```diff
@@ -30,6 +30,7 @@ buildscript {
         junit="4.12"
         runner="1.1.1"
         espresso_core="3.1.1"
+        navigation="1.0.0-alpha01"
```

Comment:
are we gonna use navigation, shouldn't we discuss it first? the benefits and problems?